stdlib/2.7/logging/__init__.pyi

```diff
@@ -4,14 +4,14 @@
 
 from typing import Any, Dict, Optional, Sequence, Tuple, overload, Union
 
-CRITICAL = 0
-FATAL = 0
-ERROR = 0
-WARNING = 0
-WARN = 0
-INFO = 0
-DEBUG = 0
-NOTSET = 0
+CRITICAL = 0 # type: int
```

Comment:
That's true. I thought, from what I read in the other stubs that `CRITICAL = ... # type: int` was the preferred way to go.